source/agora/flash/Channel.d

```diff
@@ -94,7 +94,8 @@ public class Channel
 
     /// Called when the channel state has been changed
     private alias OnChannelNotify = void delegate (
-        PublicKey key, Hash chan_id, ChannelState state, ErrorCode error) @safe;
+        PublicKey key, Hash chan_id, ChannelState state, ErrorCode error,
+        Height height = Height(0)) @safe;
```

Comment:
Same question about the default value

Reply:
Not all events are associated with a height, thus the optional argument.